core/src/test/java/io/questdb/cairo/AbstractCairoTest.java

```diff
@@ -155,7 +155,8 @@ public long getWriterAsyncCommandMaxTimeout() {
             public int getSqlJitMode() {
                 // JIT compiler is a beta feature and thus is disabled by default,
                 // but we want to have it enabled in tests.
-                return SqlJitMode.JIT_MODE_ENABLED;
+                //return SqlJitMode.JIT_MODE_ENABLED;
+                return SqlJitMode.JIT_MODE_DISABLED;
```

Comment:
Lets get to the bottom of the crash.

For context with JIT enabled this test crashes on JDK17